### PR TITLE
implemented http standard menu items

### LINKED ISSUE #283: http, missing standard menu items
Should be using the same menu items found in Android and the other interfaces.

### PR DESCRIPTION
**Pull request by Sirmorrison**
fixes #283
image
image

### CODE REVIEW

File: web/routes/walletloader.go [hunk 1]
```diff
@@ -42,19 +42,19 @@ func (routes *Routes) walletLoaderFn(next http.Handler) http.Handler {
...
-		case walletcore.SyncStatusNotStarted:
-			errMsg = "Cannot display page. Blockchain hasn't been synced"
-		case walletcore.SyncStatusInProgress:
-			errMsg = fmt.Sprintf("%s. Refresh after a while to access this page", routes.blockchain.report())
-		case walletcore.SyncStatusError:
-			errMsg = fmt.Sprintf("Cannot display page. %s", routes.blockchain.report())
-		default:
-			errMsg = "Cannot display page. Blockchain sync status cannot be determined"
-		}
+		// currentSyncStatus := routes.blockchain.status()
```
Comment (ademuanthony) [inline web/routes/walletloader.go]: Why did you remove wallet opening logic

**Review by ademuanthony — DISMISSED** [review]

File: web/views/accounts.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                       <h5 class="card-title">Accounts</h5>
+                       <p class="lead-text">Accounts page coming soon</p>
+                   </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    {{ template "footer" }}
+    <style>
+        .btn-form-group {
```
Comment (itswisdomagain) [inline web/views/accounts.html]: I can't see the elements being styled here. Where are the `.btn-form-group` and `.result-card` elements?

File: web/routes/routes.go [hunk 3]
```diff
@@ -90,4 +90,6 @@ func (routes *Routes) registerRoutesRequiringWallet(router chi.Router) {
 	router.Get("/transaction-details/{hash}", routes.transactionDetailsPage)
 	router.Get("/staking", routes.stakingPage)
 	router.Post("/purchase-tickets", routes.submitPurchaseTicketsForm)
+	router.Get("/security", routes.securityPage)
+	router.Get("/accounts", routes.accountsPage)
```
Comment (itswisdomagain) [inline web/routes/routes.go:93]: The nav menu order has accounts before security. Let's maintain that here.

File: web/routes/handlers.go [hunk 4]
```diff
@@ -524,3 +524,13 @@ func (routes *Routes) updateSetting(res http.ResponseWriter, req *http.Request)
 
 	data["success"] = true
 }
+
+func (routes *Routes) securityPage(res http.ResponseWriter, req *http.Request) {
+	data := map[string]interface{}{}
+	routes.renderPage("security.html", data, res)
+}
+
+func (routes *Routes) accountsPage(res http.ResponseWriter, req *http.Request) {
```
Comment (itswisdomagain) [inline web/routes/handlers.go]: The nav menu order has accounts before security. Let's maintain that here. This `accountsPage` function should come before `securityPage`

File: web/views/layout.html [hunk 5]
```diff
@@ -66,6 +66,21 @@ <h3>GoDCR <small data-target="connection-info.networkType">{{ .NetworkType }}</s
...
+                        <a class="nav-link" id="nav-accounts" href="/accounts">
+                            <span class="text">Accounts</span>
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" id="nav-security" href="/security">
+                            <span class="text">Security</span>
+                        </a>
+                    </li>
+                    <li class="nav-item">
```
Comment (itswisdomagain) [inline web/views/layout.html]: We have another nav item for settings just below, this isn't needed.

**Review by itswisdomagain — CHANGES_REQUESTED** [review]

**Review by itswisdomagain — APPROVED** [review]

**Pull request merged**